src/common/libs/bCMS/bCMS.php

```diff
@@ -164,80 +164,34 @@ function s3URL($fileid, $forceDownload = false, $expire = '+10 minutes', $shareK
 
     $instanceIgnore = false;
     $secure = true;
+
     // This list is also used to populate the files deletion suggestor
+    // see https://adam-rms.com/docs/v1/contributor/files for full file types
     switch ($file['s3files_meta_type']) {
       case 2:
-        $instanceIgnore = true;
```

Comment:
Not sure if just displaying weirdly - but seems to be missing the instance ignore for 2

Reply:
Should fall through in the switch case?

Reply:
But don't we need instance ignore to be true for 2

Reply:
It is in the final case: https://github.com/adam-rms/adam-rms/blob/3949d3ca7abd91ab49f41eb42d8277b61586279c/src/common/libs/bCMS/bCMS.php#L171